docs/intro.md

```diff
@@ -1 +1,42 @@
-**Telescope ALBA** documentation.
+- Assume that a prover has a large collection of data. They want to convince a verifier that their set contains at least a minimum number of elements satisfying a specific condition, known as a predicate. This remains true even if the prover only shares a portion of their data.
```

Comment:
This paragraph seems to be very similar to the opening paragraph of `README.md` - is this intentional?